app/assets/src/components/views/PipelineViz/PipelineViz.jsx

```diff
@@ -733,21 +786,32 @@ class PipelineViz extends React.Component {
     });
 
     return (
-      <div>
-        <PanZoom
-          className={cs.panZoomContainer}
-          minZoom={zoomMin}
-          maxZoom={zoomMax}
-          zoomSpeed={3}
-          ref={this.panZoomContainer}
-        >
-          <div className={cs.pipelineViz}>{stageContainers}</div>
-        </PanZoom>
-        <PlusMinusControl
-          onPlusClick={this.panZoomContainer && this.panZoomContainer.zoomIn}
-          onMinusClick={this.panZoomContainer && this.panZoomContainer.zoomOut}
-          className={cs.plusMinusControl}
-        />
+      <div className={cs.pipelineVizPage}>
+        {this.renderHeader()}
+        <div className={cs.pipelineVizContainer}>
+          <PanZoom
+            className={cs.panZoomContainer}
+            minZoom={zoomMin}
+            maxZoom={zoomMax}
+            zoomSpeed={3}
+            ref={this.panZoomContainer}
+          >
+            <div className={cs.pipelineViz}>{stageContainers}</div>
+          </PanZoom>
+          <PlusMinusControl
+            onPlusClick={
+              this.panZoomContainer &&
```

Comment:
Shorter with: `((this.panZoomContainer || {}).current || {}).zoomIn` or with `lodash/fp` `get`: `get('current.zoomIn', this.panZoomContainer)`